tensorflow/models/image/cifar10/cifar10.py

```diff
@@ -256,7 +256,8 @@ def inference(images):
     local4 = tf.nn.relu(tf.matmul(local3, weights) + biases, name=scope.name)
     _activation_summary(local4)
 
-  # softmax, i.e. softmax(WX + b)
+  # linear layer(WX + b),
+  # no softmax ops here because tf.nn.sparse_softmax_cross_entropy_with_logits() only accept unscaled logits.
```

Comment:
"We don't apply softmax here because tf.nn.sparse_softmax_cross_entropy_with_logits accepts the unscaled logits and performs the softmax internally for efficiency"

Also, please wrap lines to 80 columns.  Thanks!